website/docs/timeline.md

```diff
@@ -3,40 +3,386 @@ title: Timeline
 toc: true
 ---
 
-## Timeline
-At its core, Hudi maintains a `timeline` of all actions performed on the table at different `instants` of time that helps provide instantaneous views of the table,
-while also efficiently supporting retrieval of data in the order of arrival. A Hudi instant consists of the following components
+A Hudi table maintains all operations happened to the table in a single timeline comprised of two parts, an active timeline and an archived timeline. The active timeline stores all the recent instants, while the archived timeline stores the older instants. An instant is a transaction where all respective partitions within a base path have been successfully updated by either a writer or a table service. Instants that get older in the active timeline are moved to archived timeline at various times.
 
-* `Instant action` : Type of action performed on the table
```

Comment:
these are very foundational concepts to explain the core design. Not sure omitting them is a good idea

Reply:
not sure if I follow- it's on the doc. Just further down. 

<img width="849" alt="Screenshot 2023-03-23 at 10 22 42 PM" src="https://user-images.githubusercontent.com/5392555/227431653-6fb601ed-8d72-4604-8c0a-0f04f529a90e.png">